src/routes/team/[team]/[env]/secret/[secret]/+page.svelte

```diff
@@ -0,0 +1,279 @@
+<script lang="ts">
+	import { page } from '$app/stores';
+	import {
+		Alert,
+		Button,
+		CopyButton,
+		Heading, HelpText,
+		Loader,
+		Table, Tbody, Tooltip
+	} from '@nais/ds-svelte-community';
+	import type { PageData } from './$houdini';
+	import Card from '$lib/Card.svelte';
+	import { graphql, type Secret$result } from '$houdini';
+	import {
+		filterAddKvs,
+		mergeChanges,
+		type operation,
+		type updateState
+	} from './state-machinery';
+	import Confirm from '$lib/components/Confirm.svelte';
+	import SecretKv from './SecretKv.svelte';
+	import AddSecretKv from './AddSecretKv.svelte';
+	import { goto } from '$app/navigation';
+
+	export let data: PageData;
+	$: ({ Secret } = data);
+
+	$: secret = $page.params.secret;
+	$: env = $page.params.env;
+	$: team = $page.params.team;
+
+	let changes: operation[];
+	$: changes = [];
```

Comment:
```suggestion
	let changes: operation[] = [];
```

Denne er en noop. `$:` evaluerer dette hver gang høyresiden av `=` endres, og siden det ikke er noe dynamisk på den siden, så holder det med `let` (som er dynamiske som default i Svelte 4, dersom de er definert på root av script)